### DISCUSSION TITLE
Project going forward, suggestions

### DISCUSSION THREAD

**Discussion by brunogcar** [Mikubill/sd-webui-controlnet#353] [General]
some ideas to help in future development

1.  wiki - explaining features, in more extensive way that [README.md](https://github.com/Mikubill/sd-webui-controlnet#readme) does, in details for each with detailed examples, etc and even troubleshooting for regular problems, that would help with 2 and especially 3, also im sure  collaborators would help to edit the wiki and expand knowledge, there is already great videos around 
2.  guidelines to issues - asking to tell OS, browser, provide logs, etc to fast track lots of problems, telling ppl to search if there is already a similar issue and so on, also encouraging people do open discussions instead of issues, lots of issues are just ideas, not even feature requests
3.  actually close issues - there are many open issues that were resolved with commits, lots of duplicates, many others that are just non related, not to mention non sense issues to control net, im sure many ppl are capable of being contributors that way, with current pace, issues number will be unbearable,  sometimes, just answer and close, as there is not much to discuss, or if any relevant info is provided further they can reopen or create new issue
4.  developers - over time ppl you trust, and know the code like @lllyasviel can commit to the repo, at fist may pull request then merge if its ok, same to his repo for example, where you both share same structure and colab with each other, but many more may come , i kinda find strange that projects over this community are basicly solo projects

i believe that with those measures things will be easier for you and for ppl that follow development

**Comment by Mikubill** [comment]
Nice idea! Having more collaborators would be a great help.

**Reply by ljleb** [reply]
Hey I was recently added as a collaborator to the repo, but I'm not sure what I'm supposed to do to be honest. Was it in reaction to this discussion? People with the collaborator role seem to have permissions to edit/close issues and merge/edit/close PRs for example, but I'm not sure which of these you feel comfortable for me to do. I tried to take useful actions these last days, maybe taking a bit of the weight of the tasks that need attention off your shoulders, but I'm not confident I am really adding much value to the code or discussions.

If there's anything I have been doing that you want me to stop or change, let me know. Also, feel free to revert the decision to include me as a collaborator if you believe I am harming the pace of the project or lacking too many skills to be relevant. (I really don't know much about ml transformers! I wish I did)

**Reply by Mikubill** [reply]
Take it easy, you've done very great - just handle the issues and discussions, add features or create new issues when you come up with ideas. I'll also consider adding more collaborators later if needed, for now I think current status is appropriate.

btw, you may notice that development of this extension is slightly slower than the pace at which this repo started - give me some time to do my chores :)

**Comment by ljleb** [comment]
I think another thing this repo needs that would go in hand with your point 2, is issue templates. This would make it so that bug reports duplications are kept to a minimum and we have to ask for more information less often.
